django/db/migrations/autodetector.py

```diff
@@ -1080,11 +1080,31 @@ def generate_removed_fields(self):
             self._generate_removed_field(app_label, model_name, field_name)
 
     def _generate_removed_field(self, app_label, model_name, field_name):
+        field = self.from_state.models[app_label, model_name].get_field(field_name)
+        # You can't just remove NOT NULL fields with no default or fields
```

Comment:
Please avoid the use of "personalised" statements "You can't just…", use direct, declarative statements "NOT NULL fields can't …" 

https://docs.djangoproject.com/en/4.2/internals/contributing/writing-code/coding-style/#python-style